rules/do-you-know-how-to-delete-orphaned-users-from-the-people-picker/rule.md

```diff
@@ -11,20 +11,19 @@ redirects: []
 
 ---
 
-The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory. 
+The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory.
 <!--endintro-->
 ![](PeoplePicker.jpg)
-**Figure: The People Picker** 
+**Figure: The People Picker**
 Unfortunately, if you have added users directly to your site collection, and later deleted or disabled them from Active Directory, you will notice that these orphaned users will still appear in your People Picker. This will eventually clutter up your People Picker.
 ![](PeoplePicker-deleted.jpg)
 **Figure: A user in the People Picker that was deleted from Active Directory** ![](PeoplePicker-searching.jpg)
-**Figure: Searching in Active Directory for the user shows the user is in fact deleted** 
-To remove these orphaned users, as a SharePoint Administrator, you can open the following URL (where www.northwind.com is your SharePoint URL):
-
-* http://www.northwind.com/\_catalogs/users/simple.aspx
+**Figure: Searching in Active Directory for the user shows the user is in fact deleted**
+To remove these orphaned users, as a SharePoint Administrator, you can open the following URL (where <www.northwind.com> is your SharePoint URL):
 
+* ```http://www.northwind.com/\_catalogs/users/simple.aspx```
```

Comment:
```suggestion
* <http://www.northwind.com/\_catalogs/users/simple.aspx> 
```
Should this be like this?

Reply:
Hi @tombui99

Markdown - When you have a 1 line of code... instead of a block... use 1 ` only instead of 3

 

- [ ] 1. Please fix (I think using URL in a code is the way to go... but use @JackDevAU 's suggestion if it works well)

- [ ]  2. Change from "simple" to "sample"

Reply:
Hey @JackDevAU 

If we use "< >" then it will still be displayed as a link :( So I believe going for single ` is the way to go

Hey @tiagov8 

> 1. Please fix (I think using URL in a code is the way to go... but use @JackDevAU 's suggestion if it works well)
 2. Change from "simple" to "sample"

All done